MdePkg/Include/Base.h

```diff
@@ -206,6 +206,27 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define PACKED
 
+// Omit Function from Stack Overflow Protection.
+#ifndef NO_STACK_COOKIE
+  #if defined (__clang__)
+    #if __has_attribute (no_stack_protector)
+#define NO_STACK_COOKIE  __attribute__ ((no_stack_protector))
+    #else
+#define NO_STACK_COOKIE

```

Comment:
It seems to me that these `#define `NO_STACK_COOKIE`` cases are just the code is going to crash? It basically means the C code that requires NO_STACK_COOKIE is not going to work for these compilers?

Reply:
Yes. Let me get more specific details when I have access to my notes.

Reply:
Resolving as we are dropping the dynamic implementation for now.